modules/panorama/README.md

```diff
@@ -6,47 +6,30 @@ A terraform module for deploying a working Panorama instance in Azure.
 
 ```hcl
 module "panorama" {
-  source = "github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules//modules/panorama"
+  source  = "PaloAltoNetworks/vmseries-modules/azurerm//modules/panorama"
+  version = "0.1.0"
```

Comment:
We can now use the Registry syntax. It would be best to update all the snippets with this one. The previous github syntax has no version, which is horribly unstable.

Reply:
👍

...but ;-) in examples related to a single module only, maybe we could use "latest" tag or sth like this?

Reply:
Ummm... I'd rather not?